identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/screen/editProfile/dialog/GetImageDialog.kt

```diff
@@ -0,0 +1,131 @@
+package net.thechance.mena.identity.presentation.screen.editProfile.dialog
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import mena.identity_presentation.generated.resources.Res
+import mena.identity_presentation.generated.resources.camera
+import mena.identity_presentation.generated.resources.image
+import mena.identity_presentation.generated.resources.image_remove
+import net.thechance.mena.designsystem.presentation.component.dialog.BasicDialog
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.scaffold.Scaffold
+import net.thechance.mena.designsystem.presentation.component.scaffold.ScaffoldScope
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.MenaTheme
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import org.jetbrains.compose.resources.painterResource
+import org.jetbrains.compose.ui.tooling.preview.Preview
+
+@Composable
+fun ScaffoldScope.GetImageDialog(
+    isVisible: Boolean,
+    onDismiss: () -> Unit,
+    onUploadImage: () -> Unit,
+    onTakeImageFromCamera: () -> Unit,
+    onRemoveImage: () -> Unit,
+) {
+    BasicDialog(
+        isVisible = isVisible,
+        onDismiss = onDismiss
+    ) {
+        Column(
+            modifier = Modifier
+                .align(Alignment.Center)
+                .padding(vertical = 12.dp),
```

Comment:
We can use theme spacing
```suggestion
                .padding(vertical = Theme.spacing._12),
```